adoc/extensions/sycl_khr_max_work_group_query.adoc

```diff
@@ -0,0 +1,137 @@
+[[sec:khr-max-work-group-query]]
+= sycl_khr_max_work_group_query
+
+This extension allows developers to query iteration bounds in each dimension and
+in total for an ND-range kernel.
+The application must ensure that the number of work-groups in each dimension
+[code]#N# of an ND-range kernel is less than or equal to
+[code]#max_work_group_range<N>#.
```

Comment:
```suggestion
The application must ensure that the number of work-groups of an ND-range kernel is within the range of values returned by these queries.
```

I think we should make this sentence a bit more generic.  Maybe something like this?  (Note you will need to insert a line break at 80 columns.)